### PR TITLE
feature/add-imbuildings-sensors

### PR DESCRIPTION
**Pull request by J2thatsme**
Uplinkscript is the same one for all. Office Occupancy & People Counter behave the same way

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
   },
   "rules": {
     "no-bitwise": "off",
+    "no-plusplus": "off",
```
Comment (mariusgiger) [inline .eslintrc.json:20]: why is this turned off? use i += 1; instead

File: types/IMBuilding/ComfortSensor/meta.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Comfort sensor",
+  "version": "1.0.0",
+  "manufacturer": "IMBuildings",
```
Comment (mariusgiger) [inline types/IMBuilding/ComfortSensor/meta.json:4]: change the root directory to match the name of the manufacturer > IMBuildings

File: types/IMBuilding/OfficeOccupancy/uplink copy.js [hunk 3]
```diff
@@ -0,0 +1,277 @@
+const payloadTypes = {
```
Comment (mariusgiger) [inline types/IMBuilding/OfficeOccupancy/uplink copy.js:1]: remove this unused file

File: types/IMBuilding/OfficeOccupancy/uplink.js [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+            e: readUInt16BE(bytes, bytes.length - 2),
+          };
+          break;
+        default:
+          break;
+      }
+      emit("sample", { data, topic: "default" });
+      break;
+    }
+    default:
```
Comment (mariusgiger) [inline types/IMBuilding/OfficeOccupancy/uplink.js:104]: consider throwing an exception in the default case (unsupported payload)

**Review by mariusgiger — CHANGES_REQUESTED** [review]
looks good, a few minor comments

File: .eslintrc.json (hunk 1 above)
Comment (J2thatsme) [reply .eslintrc.json:20]: The explanation on why we should not use ++ is, that the code gets "cryptic". But as i always use it on a single line, it should not lead to errors or complicated code. Excerpt from [stack overflow](https://stackoverflow.com/questions/3000276/the-unexpected-error-in-jslint). Also the IDE generates loops with ++ so should we still go with += 1 ?

Comment (mariusgiger) [reply .eslintrc.json:20]: refer to https://eslint.org/docs/rules/no-plusplus, it makes sense to not use the operator

File: types/IMBuilding/ComfortSensor/meta.json (hunk 2 above)
Comment (mariusgiger) [reply types/IMBuilding/ComfortSensor/meta.json]: the root dir is still not changed

**Review by mariusgiger — APPROVED** [review]

**Pull request merged**